example/exampleUser/ExampleUserProvider.js

```diff
@@ -58,16 +58,21 @@ const STATUSES = [{
  * @implements {StatusUserProvider}
  */
 export default class ExampleUserProvider extends EventEmitter {
-    constructor(openmct, {defaultStatusRole} = {defaultStatusRole: undefined}) {
+    constructor(openmct, {statusRoles} = {
+        statusRoles: []
+    }) {
         super();
 
         this.openmct = openmct;
         this.user = undefined;
         this.loggedIn = false;
         this.autoLoginUser = undefined;
-        this.status = STATUSES[0];
+        this.statusRoleValues = statusRoles.map(x => ({
```

Comment:
we should use a better variable name than `x`, maybe `role`?

Reply:
Definitely open to that. I was trying to avoid a `role.role === role` type scenario that happens in the codebase.